ui/tests/integration/components/masked-input-test.js

```diff
@@ -96,4 +96,26 @@ module('Integration | Component | masked input', function (hooks) {
     const unMaskedValue = document.querySelector('.masked-value').value;
     assert.strictEqual(unMaskedValue, this.value);
   });
+
+  test('it renders correctly with empty string provided as value', async function (assert) {
```

Comment:
I would love to add an additional 2 tests to check if the correct flash message (success or danger) pops up when the value is copied, but I tried for a while and couldn't seem to get a test to work (when I pause the test and click the copy button, no flash message pops up at all). The copy button is working for MaskedInput when it's not in a test, though. Please let me know if it's worth trying again to add these 2 tests or if this PR seems fine without it!